diff --git a/src/RevealBase.js b/src/RevealBase.js
index b7df7615c4cbc6ef50c55ae23b3f415a5ae14b7a..80694fe6014862769f1e3eddca7f26459e2ebcdf 100644
--- a/src/RevealBase.js
+++ b/src/RevealBase.js
@@ -370,7 +370,7 @@ class RevealBase extends React.Component {
     };
   }
 
-  componentWillReceiveProps (props) {
+  UNSAFE_componentWillReceiveProps (props) {
     if (props.when !== undefined)
       this.isOn = !!props.when;
     if (props.fraction !== this.props.fraction)